exchanges/btse/btse_types.go

```diff
@@ -2,9 +2,23 @@ package btse
 
 import "time"
 
+// OverviewData stores market overview data
+type OverviewData struct {
+	High24Hr         string `json:"high24hr"`
```

Comment:
Please convert strings to their appropriate types ie this would be more convenient in this application as a `float64`. Please review all of these types